data/data/baremetal/masters/variables.tf

```diff
@@ -33,3 +28,18 @@ variable "instance_infos" {
   type        = list(map(string))
   description = "Instance information for hosts"
 }
+
+variable "master_ignition_url" {
```

Comment:
Sorry for the confusion around the naming. These didn't **need** to change, as you correctly stated that there is already context in the path. You can leave it as is or change it back: Your call.

Reply:
@staebler since CI passed I'm fine to leave it with the master prefix - it also makes it slightly more explicit when debugging and looking at the tfvars directly I guess